manifests/oned/sunstone/config.pp

```diff
@@ -23,26 +23,28 @@
   $tmpdir             = $one::sunstone_tmpdir,
 ){
   File {
-    ensure  => 'file',
     owner   => 'root',
-    group   => 'oneadmin',
   }
   file { '/usr/lib/one/sunstone':
-    ensure  => 'directory',
+    ensure  => directory,
+    # not sure if this is necessary
     owner   => 'oneadmin',
+    group   => 'oneadmin',
```

Comment:
please set file resource type mode explicitly unless you are absolutely sure that any mode is irrelevant for service functionality.